.github/workflows/tests.yml

```diff
@@ -14,18 +14,28 @@ concurrency:
 
 jobs:
   run-tests:
-    strategy:
-      fail-fast: false
-      matrix:
-        test-subset: ["turtle", "not turtle"]
-    runs-on: ubuntu-latest
-    name: Run pyjanitor test suite
+    name: ${{ matrix.os }}, ${{ matrix.python-version }}, ${{ matrix.env }}
+    runs-on: ${{ matrix.os }}
 
     # https://github.com/marketplace/actions/setup-miniconda#use-a-default-shell
     defaults:
       run:
         shell: bash -l {0}
 
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        dev: [false]
+        python-version:
+          - "3.8"
+          - "3.9"
+          - "3.10"
+        env: [ci/envs/latest.yaml]
```

Comment:
@Zeroto521 would you recommend also testing on the development environment as defined in `environment-dev.yml`?